### PR TITLE
ExportToTiff now logs warning and finishes instead of failing.

### PR DESCRIPTION
**Pull request by veseln**
If the _ExportToTiff_ task is requested to create a map for a feature
that is not present in the EOPatch it will exit without an error and
log a warning message instead of exiting with an exception.

This is useful for workflows where we want to export multiple features
over a set of EOPatches. In the previous version the workflow would
fail and stop processing other features once it encountered one that
was not present in the EOPatch. This fix changes the behaviour so that
encountering a missing feature does not stop the execution for that
eopatch.

### CODE REVIEW

File: io/eolearn/io/local_io.py [hunk 1]
```diff
@@ -161,7 +164,12 @@ def execute(self, eopatch, *, filename=None):
         :return: Unchanged input EOPatch
         :rtype: EOPatch
         """
-        feature_type, feature_name = next(self.feature(eopatch))
+        try:
+            feature_type, feature_name = next(self.feature(eopatch))
+        except ValueError as error:
+            LOGGER.warning(error)
```
Comment (devisperessutti) [inline io/eolearn/io/local_io.py]: is this error explicit enough for the user to understand that the feature is missing? 

or should we add a string that does that?

**Comment by devisperessutti** [comment]
> # [Codecov](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=h1) Report
> 
> > Merging [#158](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=desc) into [develop](https://codecov.io/gh/sentinel-hub/eo-learn/commit/3f789e9a2306994e33701cab606f131dc6f5904d?src=pr&el=desc) will **decrease** coverage by `0.03%`.
> > The diff coverage is `57.14%`.
> 
> [Impacted file tree graph](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=tree)
> 
> ```diff
> @@             Coverage Diff             @@
> ##           develop     #158      +/-   ##
> ===========================================
> - Coverage    78.61%   78.57%   -0.04%     
> ===========================================
>   Files           42       42              
>   Lines         4507     4513       +6     
> ===========================================
> + Hits          3543     3546       +3     
> - Misses         964      967       +3
> ```
> 
> [Impacted Files](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=tree) 	Coverage Δ 	
> [io/eolearn/io/local_io.py](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158/diff?src=pr&el=tree#diff-aW8vZW9sZWFybi9pby9sb2NhbF9pby5weQ==) 	`90% <57.14%> (-1.8%)` 	arrow_down
> 
> [Continue to review full report at Codecov](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=continue).
> 
> > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > Powered by [Codecov](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=footer). Last update [3f789e9...e8af314](https://codecov.io/gh/sentinel-hub/eo-learn/pull/158?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

could we add a simple test [here](https://github.com/sentinel-hub/eo-learn/blob/3f789e9a2306994e33701cab606f131dc6f5904d/io/eolearn/tests/test_local_io.py#L152) to check the expected behaviour ?

File: io/eolearn/io/local_io.py (hunk 1 above)
Comment (veseln) [reply io/eolearn/io/local_io.py]: Yes, the error is very explicit. Example: 

> WARNING eolearn.io.local_io:170: Feature VALID_COUNT_MULTI_150m of type FeatureType.MASK_TIMELESS was not found in EOPatch

Comment (devisperessutti) [reply io/eolearn/io/local_io.py]: perfect

**Pull request merged**